atlassian/confluence.py

```diff
@@ -2798,17 +2800,18 @@ def get_atl_request(url: str):
             parsed_html = BeautifulSoup(response.text, "html.parser")
             # Getting the poll URL to get the export progress status
             try:
-                poll_url = parsed_html.find("meta", {"name": "ajs-pollURI"}).get("content")
+                poll_url = cast("str", parsed_html.find("meta", {"name": "ajs-pollURI"}).get("content"))  # type: ignore[union-attr]
```

Comment:
These got ugly in mypy quickly.  `parsed_html.find` could return `Tag`, `NavigableString`, or `None`; and only `Tag` supports `get()`.   
cast as `str` to avoid mypy error on line 2808.